### PR TITLE
/search endpoint completed

### PR DESCRIPTION
**Pull request by YoginiTayade**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-CRED_URL = {{ credentials.CRED_URL }}
-DID_URL = {{ credentials.DID_URL }}
-SCHEMA_URL = {{ credentials.SCHEMA_URL }}
-KEYCLOAK_REALM_ID= {{ credentials.KEYCLOAK_REALM_ID }}
-KEYCLOAK_CLIENT_ID= {{ credentials.KEYCLOAK_CLIENT_ID }}
-KEYCLOAK_CLIENT_SECRET= {{ credentials.KEYCLOAK_CLIENT_SECRET }}
-KEYCLOAK_URL= {{ credentials.KEYCLOAK_URL }}
-REGISTRY_URL= {{ credentials.REGISTRY_URL }}
-TELEMETRY_URL= {{ credentials.TELEMETRY_URL }}
+TESTVAR= development environment
```
Comment (rushi-tekdi) [inline .env:1]: @YoginiTayade discard .env changes and then commit code

File: src/claimAttest/claimAttest.controller.ts [hunk 2]
```diff
@@ -27,9 +27,11 @@ export class ClaimAttestController{
             credentialSubject,
             response);
     }
-    @Post('/search')
-    search(){
-        return this.claimAttestService.search();
+    @Get('/search')
+    search(@Headers('Authorization') token:string,
```
Comment (rushi-tekdi) [inline src/claimAttest/claimAttest.controller.ts:31]: @YoginiTayade  Add Async before all clientAttest controller functions. Refer other apis controller

File: src/claimAttest/claimAttest.service.ts [hunk 3]
```diff
@@ -171,10 +170,138 @@ export class ClaimAttestService {
...
+              eq: gender,
+            },
+          },
+        };
+
+        const userDetails = await this.sbrcService.sbrcSearch(
+          searchSchema,
+          'Instructor',
+        );
+        if (userDetails[0]?.school_id) {
```
Comment (rushi-tekdi) [inline src/claimAttest/claimAttest.service.ts:258]: @YoginiTayade  Add else part of this if 
in else part add "school id not found in instructor account" with status: false 400

File: src/claimAttest/claimAttest.service.ts [hunk 4]
```diff
@@ -171,10 +170,138 @@ export class ClaimAttestService {
...
+              message: 'Data not present in the system.',
+              result: sbrcSearch,
+            });
+          }
+        } else {
+          return response.status(404).send({
+            success: false,
+            status: 'search_fail',
+            message: 'Data not found in the system.',
+            result: userDetails[0]?.school_id,
```
Comment (rushi-tekdi) [inline src/claimAttest/claimAttest.service.ts:296]: @YoginiTayade  result:null

File: .env (hunk 1 above)
Comment (YoginiTayade) [reply .env]: resolved

File: src/claimAttest/claimAttest.service.ts (hunk 3 above)
Comment (YoginiTayade) [reply src/claimAttest/claimAttest.service.ts:258]: resolved

File: src/claimAttest/claimAttest.service.ts (hunk 4 above)
Comment (YoginiTayade) [reply src/claimAttest/claimAttest.service.ts]: resolved

File: .env (hunk 1 above)
Comment (rushi-tekdi) [reply .env:1]: @YoginiTayade  .env file has issue
add below in that file and push code again
TESTVAR= {{ credentials.TESTVAR }}
PORT = {{ credentials.PORT }}
CRED_URL = {{ credentials.CRED_URL }}
DID_URL = {{ credentials.DID_URL }}
SCHEMA_URL = {{ credentials.SCHEMA_URL }}
KEYCLOAK_REALM_ID= {{ credentials.KEYCLOAK_REALM_ID }}
KEYCLOAK_CLIENT_ID= {{ credentials.KEYCLOAK_CLIENT_ID }}
KEYCLOAK_CLIENT_SECRET= {{ credentials.KEYCLOAK_CLIENT_SECRET }}
KEYCLOAK_URL= {{ credentials.KEYCLOAK_URL }}
REGISTRY_URL= {{ credentials.REGISTRY_URL }}
TELEMETRY_URL= {{ credentials.TELEMETRY_URL }}
VERIFICATION_URL= {{ credentials.VERIFICATION_URL }}
LEARNER_SCHEMA_FIELD= {{ credentials.LEARNER_SCHEMA_FIELD }}

**Pull request merged**